### PR TITLE
Move wip checks to github actions

### PR DESCRIPTION
**Pull request by jameesjohn**
## Explanation
Moves WIP/Draft checks to github actions

These features have been tested at https://github.com/jameesjohn/comment-on-pr/pull/51#issuecomment-866029967

## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: actions/src/dispatcher.js [hunk 1]
```diff
@@ -49,6 +50,9 @@ module.exports = {
             case constants.dontMergeLabelCheck:
               await PRLabelsModule.checkUnLabeled();
               break;
+            case constants.wipCheck:
+              await wipDraftModule.checkWIP();
```
[the remark this answers is not in the document]
Comment (jameesjohn) [reply actions/src/dispatcher.js]: Done.

File: actions/src/utils.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const { context } = require('@actions/github');
+
+/**
+ * This function pings and assigns all pending reviewers to a pull request.
```
Comment (vojtechjelinek) [inline actions/src/utils.js:4]:
```suggestion
 * This function creates a comment and assigns users.
```

File: actions/src/utils.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ * @param {import('@octokit/rest').Octokit} octokit
+ * @param {import('@octokit/rest').Octokit.PullsGetResponse} pullRequest
+ * @param {string[]} assignees
+ * @param {string} comment
+ */
+const pingAndAssignUsers = async (octokit, pullRequest, assignees, comment) => {
```
Comment (vojtechjelinek) [inline actions/src/utils.js:10]: This function doesn't really ping users, it just creates comment (which might contain entirely different content).

File: actions/src/pull_requests/checkWipDraftPR.js [hunk 4]
```diff
@@ -33,50 +40,50 @@ const isWIPPr = ({ title, body }) => {
...
+      const commentBody =
+        'Hi @' +
+        prAuthor +
+        ', when creating WIP/Draft PRs, ensure that ' +
         'your commit messages are prefixed with **[ci skip]** or ' +
         '**[skip ci]** to prevent CI checks from running. ' +
-        'You can learn more about it ' + link + '.');
+        'You can learn more about it ' +
+        link +
+        '.';
```
Comment (vojtechjelinek) [inline actions/src/pull_requests/checkWipDraftPR.js:84]:
```suggestion
        'You can learn more about it ' + link + '.';
```

File: actions/src/pull_requests/checkWipDraftPR.js [hunk 5]
```diff
@@ -33,50 +40,50 @@ const isWIPPr = ({ title, body }) => {
...
-        '#wip--draft-pull-requests');
+          '#wip--draft-pull-requests'
+      );
 
-      const commentBody = (
-        'Hi @' + prAuthor + ', when creating WIP/Draft PRs, ensure that ' +
+      const commentBody =
+        'Hi @' +
+        prAuthor +
+        ', when creating WIP/Draft PRs, ensure that ' +
```
Comment (vojtechjelinek) [inline actions/src/pull_requests/checkWipDraftPR.js:79]:
```suggestion
        'Hi @' + prAuthor + ', when creating WIP/Draft PRs, ensure that ' +
```

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Left a few comments.

File: actions/src/utils.js (hunk 3 above)
Comment (jameesjohn) [reply actions/src/utils.js:10]: Yes, but it also assigns the users. Comments can be created directly. This function helps when we want to make the comment and also assign users at the same time.

File: actions/src/pull_requests/checkWipDraftPR.js (hunk 5 above)
Comment (jameesjohn) [reply actions/src/pull_requests/checkWipDraftPR.js]: Done.

File: actions/src/pull_requests/checkWipDraftPR.js (hunk 4 above)
Comment (jameesjohn) [reply actions/src/pull_requests/checkWipDraftPR.js]: Done.

File: actions/src/utils.js (hunk 2 above)
Comment (jameesjohn) [reply actions/src/utils.js]: Done.

**Comment by jameesjohn** [comment]
PTAL @vojtechjelinek

**Review by vojtechjelinek — APPROVED** [review]
LGTM!

**Pull request merged**